src/platform/src/Bridge/Anthropic/PlatformFactory.php

```diff
@@ -25,7 +25,7 @@
     public static function create(
         #[\SensitiveParameter]
         string $apiKey,
-        string $version = '2023-06-01',
+        ?string $version = null,
```

Comment:
What about moving this parameter at the end instead?

Reply:
Might be a solution.

But still I'm unsure it a nice pattern to duplicate the default value `'2023-06-01'` between the factory and the Model, no ?